misc/pytorch_toolkit/chest_xray_screening/chest_xray_screening/utils/exporter.py

```diff
@@ -0,0 +1,41 @@
+import torch
+import os
+import subprocess
+from .model import DenseNet121, DenseNet121Eff, load_checkpoint
+
+OPENVINO_DIR = '/opt/intel/openvino_2021'
+
+class Exporter:
+    def __init__(self, config, optimised):
+
+        self.config = config
+        self.checkpoint = config.get('checkpoint')
+        if optimised:
+            self.model = DenseNet121Eff(self.config["alpha"], self.config["beta"], self.config["class_count"])
+        else:
+            self.model = DenseNet121(class_count=3)
+
+        self.model.eval()
+        load_checkpoint(self.model, self.checkpoint)
+
+    def export_model_ir(self):
+        input_model = os.path.join(os.path.split(self.checkpoint)[0], self.config.get('model_name'))
```

Comment:
```suggestion
        input_model = os.path.join(os.path.split(self.checkpoint)[0], self.config.get('model_name_onnx'))
```
Otherwise the code tries to export "model_weights/chest_xray_screening` which does not exist.